Makefile

```diff
@@ -78,7 +78,7 @@ METADATA_VAR += CommitSHA=$(EXTRA_VERSION)
 METADATA_VAR += BaseDockerLabel=$(BASE_DOCKER_LABEL)
 METADATA_VAR += DockerNamespace=$(DOCKER_NS)
 
-GO_VER = 1.18.10
+GO_VER = 1.19.2
```

Comment:
At this point, we need 1.19.7 or 1.20.2.

Please update all references to Go version, you can look at https://github.com/hyperledger/fabric/pull/3959 for the complete set of references.